src/base/init/install.c

```diff
@@ -237,12 +253,23 @@ static void call_installdos(void)
 		free(boot_dir_path);
 		return;
 	}
-	/* symlink command.com in case someone hits Shift or F5 */
-	ret = asprintf(&system_str,
-			"ln -s ../drives/d/command.com "
-			"../drives/d/kernel.sys "
-			"\"%s\"",
-			boot_dir_path);
+	switch (dos_status) {
+		case 0:
+			/* symlink command.com in case someone hits Shift or F5 */
+			ret = asprintf(&system_str,
+					"ln -s ../drives/d/command.com "
+					"../drives/d/kernel.sys "
+					"\"%s\"",
+					boot_dir_path);
+			break;
+		case 100:
+			ret = asprintf(&system_str,
+					"ln -s ../drives/d/COMMAND.COM "
```

Comment:
stackoverflow says:
find my_root_dir -depth -exec rename 's/(.*)\/([^\/]*)/$1\/\L$2/' {} \;

It is absolutely unclear for me how 7zip can be
used as mcopy and vice-versa, but I'll learn that. :)